src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/time.js

```diff
@@ -252,7 +252,7 @@ class Legend {
     }
 
     get(category) {
-        return this.categories.find(element => element.value === category);
+        return this.categories.find(element => element.value === category) || this.createCategory(category);
```

Comment:
wouldn't `this.getClassname(category)` inside `createCategory` also be an issue for unknown categories ? I think it will return `undefined`, which will do weird things when casting it to string to put it in the DOM.

Reply:
Well no it's actually fine - it's only used for the class name, which will then be `timeline-category undefined active`, and as such results in the unknown category being rendered in the `undefined` style, so light gray. It serves the purpose and the result is correct.

Reply:
I suggest updating the method to `return this.classnames[category] || ''` to avoid the weird `undefined` class.

Reply:
Fair enough, it's cleaner like that.